coderadar-vcs/src/test/java/io/reflectoring/coderadar/adapter/ComputeContributorAdapterTest.java

```diff
@@ -45,6 +50,19 @@ public void test() {
         .containsExactly("maksim.atanasov@adesso.de");
   }
 
+  @Test
+  void contributorsNotCommitedInDateRangeAreNotAddedTest() {
+    ComputeContributorAdapter computeContributorAdapter = new ComputeContributorAdapter();
+    List<Contributor> contributors =
+        computeContributorAdapter.computeContributors(
+            folder.getAbsolutePath(),
+            new ArrayList<>(),
+            new DateRange()
+                .setStartDate(LocalDate.of(2019, 8, 5))
+                .setEndDate(LocalDate.of(2019, 8, 12)));
+    Assertions.assertThat(contributors.get(0).getDisplayName()).isEqualTo("maximAtanasov");
```

Comment:
I think this assertion is only partially correct. You do check that `maximAtanasov` committed to this project in the given date range. But you don't check that nobody else commited to this project in this date range. So to make sure `maximAtanasov` is the only one who committed to the project, maybe add an assertion like

`Assertions.assertThat(contributors.size()).isEqualTo(1);`

Of course, if we adjust the `test-repository`, we maybe have to adjust the test as well.